settings.gradle.kts

```diff
@@ -114,211 +120,88 @@ dependencyResolutionManagement {
+      version("com.google.common", "31.1-jre")
+      version("com.google.protobuf", "3.19.4")
+      version("com.hedera.hashgraph.protobuf.java.api", hapiVersion)
+      version("com.hedera.pbj.runtime", "0.5.2")
+      version("com.sun.jna", "5.12.1")
+      version("com.swirlds.base", swirldsVersion)
+      version("com.swirlds.cli", swirldsVersion)
+      version("com.swirlds.common", swirldsVersion)
+      version("com.swirlds.config", swirldsVersion)
+      version("com.swirlds.fchashmap", swirldsVersion)
+      version("com.swirlds.fcqueue", swirldsVersion)
+      version("com.swirlds.jasperdb", swirldsVersion)
+      version("com.swirlds.logging", swirldsVersion)
+      version("com.swirlds.merkle", swirldsVersion)
+      version("com.swirlds.platform", swirldsVersion)
+      version("com.swirlds.test.framework", swirldsVersion)
+      version("com.swirlds.virtualmap", swirldsVersion)
+      version("dagger", daggerVersion)
+      version("dagger.compiler", daggerVersion)
+      version("grpc.netty", grpcVersion)
+      version("grpc.stub", grpcVersion)
+      version("headlong", "6.1.1")
+      version("info.picocli", "4.6.3")
+      version("io.grpc", helidonVersion)
+      version("io.helidon.grpc.client", helidonVersion)
+      version("io.helidon.grpc.core", helidonVersion)
+      version("io.helidon.grpc.server", helidonVersion)
+      version("io.netty.handler", nettyVersion)
+      version("io.netty.transport", nettyVersion)
+      version("io.netty.transport.epoll", nettyVersion)
+      version("javax.inject", "1")
+      version("net.i2p.crypto.eddsa", "0.3.0")
+      version("org.apache.commons.codec", "1.15")
+      version("org.apache.commons.collections4", "4.4")
+      version("org.apache.commons.io", "2.11.0")
+      version("org.apache.commons.lang3", "3.12.0")
+      version("org.apache.logging.log4j", log4jVersion)
+      version("org.apache.logging.log4j.core", log4jVersion)
+      version("org.assertj.core", "3.23.1")
+      version("org.bouncycastle.pkix", bouncycastleVersion)
+      version("org.bouncycastle.provider", bouncycastleVersion)
+      version("org.eclipse.collections.api", eclipseCollectionsVersion)
+      version("org.eclipse.collections.impl", eclipseCollectionsVersion)
+      version("org.hamcrest", "2.2")
+      version("org.hyperledger.besu.datatypes", besuVersion)
+      version("org.hyperledger.besu.evm", besuVersion)
+      version("org.hyperledger.besu.secp256k1", besuNativeVersion)
+      version("org.json", "20210307")
+      version("org.junit.jupiter.api", "5.9.0")
+      version("org.junitpioneer", "2.0.1")
+      version("org.mockito", mockitoVersion)
+      version("org.mockito.junit.jupiter", mockitoVersion)
+      version("org.slf4j", "2.0.3")
+      version("org.testcontainers", testContainersVersion)
+      version("org.testcontainers.junit.jupiter", testContainersVersion)
+      version("tuweni.bytes", tuweniVersion)
+      version("tuweni.units", tuweniVersion)
```

Comment:
With this approach, versions are defined using the **Module Names** (no additional aliases; the Module Name is the alias if you want)